packages/@react-aria/menu/src/useMenuItem.ts

```diff
@@ -83,6 +83,8 @@ export function useMenuItem<T>(props: MenuItemProps, state: MenuState<T>): MenuI
           onClose();
         }
         break;
+      // Enter should close the Menu even if closeOnSelect=false
+      // https://github.com/adobe-private/react-spectrum-v3/pull/171#issuecomment-580917542
```

Comment:
I disagree with this. It's quite confusing if `closeOnSelect=false` doesn't work in all cases.

Reply:
k reverted the old commit, brought back the fix